test/unit/specs/dashboard/cd-dashboard-news.spec.js

```diff
@@ -0,0 +1,172 @@
+          { type: 'News', date: moment('2018-08-14T10:00:00.000Z'), formattedDate: '14/08/2018', link: 'blah6', title: 'blah6' },
+          { type: 'News', date: moment('2018-08-13T10:00:00.000Z'), formattedDate: '13/08/2018', link: 'blah5', title: 'blah5' },
+          { type: 'News', date: moment('2018-08-12T10:00:00.000Z'), formattedDate: '12/08/2018', link: 'blah4', title: 'blah4' },
+          { type: 'News', date: moment('2018-08-11T10:00:00.000Z'), formattedDate: '11/08/2018', link: 'blah3', title: 'blah3' },
+          { type: 'News', date: moment('2018-08-10T10:00:00.000Z'), formattedDate: '10/08/2018', link: 'blah2', title: 'blah2' }]);
+      });
+      it('should return null if news is empty', () => {
+        // ARRANGE
+        const vm = vueUnitHelper(DashboardNewsComponentWithMocks);
+        vm.news = null;
+        // vm.forums = null;
+
+        // ASSERT
+        expect(vm.allPosts).to.equal(null);
+      });
+    });
+
+    describe('computed.formattedNews', () => {
+      it('should return a formatted array of news posts', () => {
+        // ARRANGE
+        const mockFormattedNews = [
+        { type: 'News', date: moment('2018-08-09T10:00:00.000Z'), link: 'blah1', title: 'blah1' },
+        { type: 'News', date: moment('2018-08-10T10:00:00.000Z'), link: 'blah2', title: 'blah2' },
+        { type: 'News', date: moment('2018-08-11T10:00:00.000Z'), link: 'blah3', title: 'blah3' },
+        { type: 'News', date: moment('2018-08-12T10:00:00.000Z'), link: 'blah4', title: 'blah4' },
+        { type: 'News', date: moment('2018-08-13T10:00:00.000Z'), link: 'blah5', title: 'blah5' },
+        { type: 'News', date: moment('2018-08-14T10:00:00.000Z'), link: 'blah6', title: 'blah6' },
+        { type: 'News', date: moment('2018-08-15T10:00:00.000Z'), link: 'blah7', title: 'blah7' }];
+
+        const vm = vueUnitHelper(DashboardNewsComponentWithMocks);
+        vm.news = [
+        { date: '2018-08-09T10:00:00.000Z', link: 'blah1', title: { rendered: 'blah1' } },
+        { date: '2018-08-10T10:00:00.000Z', link: 'blah2', title: { rendered: 'blah2' } },
+        { date: '2018-08-11T10:00:00.000Z', link: 'blah3', title: { rendered: 'blah3' } },
+        { date: '2018-08-12T10:00:00.000Z', link: 'blah4', title: { rendered: 'blah4' } },
+        { date: '2018-08-13T10:00:00.000Z', link: 'blah5', title: { rendered: 'blah5' } },
+        { date: '2018-08-14T10:00:00.000Z', link: 'blah6', title: { rendered: 'blah6' } },
+        { date: '2018-08-15T10:00:00.000Z', link: 'blah7', title: { rendered: 'blah7' } }];
+
+        // ASSERT
+        expect(vm.formattedNews).to.deep.equal(mockFormattedNews);
+      });
+    });
+
+    describe('computed.isDisplayable', () => {
+      it('should return true if loadedPosts is true', () => {
+        // ARRANGE
+        const vm = vueUnitHelper(DashboardNewsComponentWithMocks);
+        vm.loadedPosts = true;
```

Comment:
Refered earlier to [] vs null rather than a boolean. @DanielBrierton may want to confirm if it's worth the trouble

Reply:
Gotcha. It is more simple !! I already made the changes, will push in a minute when im done running unit tests